pipeline/beam_tables.py

```diff
@@ -544,12 +544,12 @@ def _parse_received_data(self, received: Union[str, Dict[str, Any]],
             received['body'],
         'received_headers':
             _parse_received_headers(received.get('headers', {})),
-        'blockpage':
-            None,
     }
 
     if anomaly:  # check response for blockpage
-      row['blockpage'] = self.blockpage_matcher.match_page(received['body'])
+      blockpage, signature = self.blockpage_matcher.match_page(received['body'])
+      row['blockpage'] = blockpage
+      row['blockpage_signature'] = signature
```

Comment:
Let's call this page_signature. We don't need to tie it to "blockpage". It could be any well known page (like the Cloudflare ddos protection page). In fact, in some cases the signature is already not for a block page.

Reply:
done